src/Sylius/Bundle/ThemeBundle/Templating/TemplateNameParser.php

```diff
@@ -57,7 +57,7 @@ public function parse($name): TemplateReferenceInterface
         }
 
         $template = new TemplateReference(
-            $matches[1] ? $matches[1] . 'Bundle' : '',
+            $this->getBundleOrPluginName($matches[1]),
```

Comment:
This can generate a notice. You have to check the existence of the index `1`.

![image](https://user-images.githubusercontent.com/858611/50963651-93152b00-14cd-11e9-9d19-c2483e0efada.png)

Reply:
Hum, according to the regex above, you should not process this code if `$matches[1]` doesn't exist, since it's not matching in that case.

Reply:
Actually, there should always be `$matches[0-5]` if the `preg_match` returns anything `>=1` (if not, the `if` above has an early return), so we can assume that they exist. IIRC that regexp is copied from Symfony source code.